signimage/Makefile

```diff
@@ -0,0 +1,34 @@
+IMAGE_TAG_BASE ?= quay.io/chrisp262/kmod-signer
+IMAGE_TAG ?= $(shell  git log --format="%H" -n 1)
```

Comment:
Why are we using a commit sha as the tag instead of always using `latest` for example?

Reply:
That will also be automated in the postsubmit job. We get both `latest` and the SHA.

Reply:
Again the Makefile is about making debugging easier so it uses the git commit because you probably want latest to reference the latest production version. As Quentin says kmmo itself should never reference it but should use the official image.